op-service/txmgr/txmgr.go

```diff
@@ -196,8 +325,15 @@ func NewSimpleTxManager(name string, l log.Logger, cfg Config, backend ETHBacken
 	if cfg.NumConfirmations == 0 {
 		panic("txmgr: NumConfirmations cannot be zero")
 	}
+	if cfg.NetworkTimeout == 0 {
+		cfg.NetworkTimeout = 2 * time.Second
+	}
+
+	// On error, ignore. We will try to get the chainID again when used.
+	chainID, _ := backend.ChainID(context.Background())
```

Comment:
Why do we do that? Shouldn't we just let `NewSimpleTxManager` return an `error` here? If we cannot query while setting up the txmgr, something is wrong.

Reply:
Well this changes the interface of the `NewSimpleTxManager` which I _really_ would like to not touch.